jest.setup.js

```diff
@@ -50,3 +50,9 @@ jest.mock('@web3-onboard/core', () => () => ({
     get: () => mockOnboardState,
   },
 }))
+
+// to avoid failing tests in some environments
+const NumberFormat = Intl.NumberFormat
+const englishTestLocale = 'en'
+
+jest.spyOn(Intl, 'NumberFormat').mockImplementation((locale, ...rest) => new NumberFormat([englishTestLocale], ...rest))
```

Comment:
non related with this PR.

Added to avoid failing tests in some environments:
![Captura de pantalla 2022-09-21 a las 19 20 58](https://user-images.githubusercontent.com/26763673/191710214-d4931588-ee72-4add-a0c5-733595e35f7d.png)
![Captura de pantalla 2022-09-21 a las 19 22 47](https://user-images.githubusercontent.com/26763673/191710219-72ebe47c-3730-49ee-82b0-894cc6d660ec.png)